src/components/Player/PlayerStats.jsx

```diff
@@ -31,7 +31,7 @@ export const PlayerStatsCards = ({
     return <Spinner />;
   }
   return (
-    <div style={{ position: 'absolute' }}>
+    <div>
```

Comment:
Can you merge in the player header fixes branch because we're going to have a ton of merge conflicts